src/files-and-videos/generic/FileTable.jsx

```diff
@@ -61,6 +61,7 @@ const FileTable = ({
   const [isAddOpen, setAddOpen, setAddClose] = useToggle(false);
   const [selectedRows, setSelectedRows] = useState([]);
   const [isDeleteConfirmationOpen, openDeleteConfirmation, closeDeleteConfirmation] = useToggle(false);
+  const [selectedFilters, setSelectedFilters] = useState([]);
```

Comment:
Why are you adding another state variable? `DataTableContext` has a `setAllFilters` function to update the table/filter state.

Reply:
This state variable is functions as a copy of `DataTableContext`'s filter state. I thought `DataTableContext` completely resets so I needed something outside of `DataTable` to keep track of the state.

Reply:
Thank you for the clarification, I understand now.